pkg/kdump/kdump-view.jsx

```diff
@@ -41,15 +41,71 @@ import { Tooltip, TooltipPosition } from "@patternfly/react-core/dist/esm/compon
 import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
 
 import { useDialogs, DialogsContext } from "dialogs.jsx";
+import { read_os_release } from "os-release.js";
 import { fmt_to_fragments } from 'utils.jsx';
 import { show_modal_dialog } from "cockpit-components-dialog.jsx";
 import { FormHelper } from "cockpit-components-form-helper";
 import { ModalError } from 'cockpit-components-inline-notification.jsx';
 import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { ModificationsExportDialog } from "cockpit-components-modifications.jsx";
 
 const _ = cockpit.gettext;
 const DEFAULT_KDUMP_PATH = "/var/crash";
 
+const exportAnsibleTask = (settings, os_release) => {
+    const target = Object.keys(settings.targets)[0];
+    const targetSettings = settings.targets[target];
+    const kdump_core_collector = settings.core_collector;
+
+    let role_name = "linux-system-roles";
+    if (os_release.NAME === "RHEL" || os_release.ID_LIKE?.includes('rhel')) {
+        role_name = "rhel-system-roles";
+    }
+
+    let ansible = `
+---
+# Also available via https://galaxy.ansible.com/ui/standalone/roles/linux-system-roles/kdump/
+- name: install ${role_name}
+  package:
+    name: ${role_name}
+    state: present
+  delegate_to: 127.0.0.1
+  become: true
+- name: run kdump system role
+  include_role:
+    name: ${role_name}.kdump
+  vars:
+    kdump_path: ${targetSettings.path || DEFAULT_KDUMP_PATH}
+    kdump_core_collector: ${kdump_core_collector}`;
+
+    if (target === "ssh") {
+        let ssh_user;
+        if (targetSettings.server.includes('@')) {
+            ssh_user = targetSettings.server.split('@')[0];
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19430-20231121-164631-d7047285-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
This is weird -- we currently have a test case which exercises "admin@ip"..

Reply:
This does not work because coverage is collected on `self.machine` and not our cockpit machine (which is machine three). So yes, I should refector enableRepo to take a machine argument and then `createPackage` as well. I can do this in a follow up.